libs/UpdatePackage.php

```diff
@@ -157,6 +158,31 @@ public function checkPhpVersion()
         return $newMinPhpVersion;
     }
 
+    /**
+     * Check for restricted modules
+     *
+     * @return string|true TRUE - if no restriction, String - error message
+     */
+    public function checkRestrictedModules()
+    {
+        // Conditions when modules should be restricted:
+        $restrictedModules = [
+            'enterprise' => ['HumHubVersion', '>=', '1.10']
+        ];
+
+        foreach ($restrictedModules as $moduleId => $restriction) {
+            if (Yii::$app->getModule($moduleId) &&
+                $restriction[0] === 'HumHubVersion' &&
+                version_compare($this->getNewConfigValue('version'), $restriction[2], $restriction[1])) {
+                return Yii::t('UpdaterModule.base', 'This version is not supported with the legecy Enterprise Edition. Please contact support! {email}', [
```

Comment:
typo: legacy

Reply:
@dreua Thanks! Fixed in https://github.com/humhub/updater/pull/32/commits/827d4206484d8575adb1d5f795d10c090a3ae75d.